### PR TITLE
Update minegasm to Minecraft 1.20.4

### PR DESCRIPTION
**Pull request by bananasov**
This PR updates Minegasm-fabric to minecraft 1.20.4 along with some changes to the `gradle.properties` file

This is my first time updating a minecraft mod so please let me know if I did anything wrong.

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
-loader_version      = 0.14.21
-fabric_version      = 0.86.1+1.20.1
+minecraft_version=1.20.4
+yarn_mappings=1.20.4+build.3
+loader_version=0.15.7
+fabric_version=0.96.11+1.20.4
 
 #Mod properties
-mod_version         = 2.1.2
+mod_version         = 2.2.0
```
Comment (vinceh121) [inline gradle.properties]: Please use 2.1.3

File: remappedSrc/com/therainbowville/minegasm/client/ClientEventHandler.java [hunk 2]
```diff
@@ -0,0 +1,325 @@
+package com.therainbowville.minegasm.client;
```
Comment (vinceh121) [inline remappedSrc/com/therainbowville/minegasm/client/ClientEventHandler.java]: Please remove every file under remappedSrc/*

**Review by vinceh121 — CHANGES_REQUESTED** [review]
Looks good to me overall, once you've fixed the couple requests I'll happily merge your PR.

File: remappedSrc/com/therainbowville/minegasm/client/ClientEventHandler.java (hunk 2 above)
Comment (bananasov) [reply remappedSrc/com/therainbowville/minegasm/client/ClientEventHandler.java]: 👍

**Pull request merged**